h/admin/views/features.py

```diff
@@ -29,6 +32,15 @@ def features_save(request):
                 setattr(feat, attr, True)
             else:
                 setattr(feat, attr, False)
+        for cohort in request.db.query(models.FeatureCohort).all():
+            val = request.POST.get('{0}[{1}]'.format(feat.name, cohort.name))
```

Comment:
Ok, I see now why you added stuff to precent cohorts called 'everyone', 'admin' and 'staff'. I'd suggest naming the form fields in such a way that this can't cause a problem, such as `<featurename>[cohort][<cohortname>]`.